assets/js/blocks/cart-checkout/cart/empty-cart/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/editor';
+import { SHOP_URL } from '@woocommerce/block-settings';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import iconDataUri from './icon-data-uri.js';
+import './style.scss';
+
+/**
+ * Component to handle edit mode for the Cart block when cart is empty.
+ */
+const EmptyCart = ( { hidden } ) => {
+	return (
+		<div hidden={ hidden }>
```

Comment:
Took me a few minutes to grasp why we're just hiding this in the editor vs avoiding rendering it (needed because if the user does not trigger the initial state for the empty cart mode, it's contents will not get persisted to the post_content).  Do you mind adding some inline docs here to explain that so our future selves don't forget?

Reply:
Sure! Added an explanation in e69fd40e3524a85bb1662c7a837bcb674c6e170b.